ql/src/test/results/clientpositive/beeline/materialized_view_create_rewrite.q.out

```diff
@@ -63,7 +63,7 @@ numFiles	1
 numFilesErasureCoded	0
 numRows	2
 rawDataSize	408
-totalSize	474
+totalSize	#Masked#
```

Comment:
I love that is masked out. The PR at the next upgrade will contain half of those files.